AmazonChimeSDK/AmazonChimeSDK/session/MeetingSessionCredentials.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 
 /// `MeetingSessionCredentials` includes the credentials used to authenticate.
 /// the attendee on the meeting
-@objcMembers public class MeetingSessionCredentials: NSObject {
+@objcMembers public class MeetingSessionCredentials: NSObject, Codable {
```

Comment:
why do we need `Codable`?

Reply:
Because we want to save them as data into UserDefaults